specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json

```diff
@@ -1130,11 +1156,11 @@
           "maximum": 65534,
           "minimum": 49152,
           "type": "integer",
-          "description": "The TCP port used for the handshake.",
+          "description": "The TCP port used for the handshake when establishing a connection.",
```

Comment:
When I tested the API I found that some members only have meaningful values when the session is ready. The ports are "0" while session is "Starting"

Would that be worthwhile to add to the comments?

Reply:
I think we'll patch the service not to return these in those cases (they are not required fields, so would be cleaner, and the ports being 0 are in violation of the swagger specification)